### PR TITLE
identify order call api implemented

### PR DESCRIPTION
**Pull request by creative-dev-lab**

### CODE REVIEW

File: Sources/AirRobeWidget/Service/Endpoints/AirRobeWidgetEndpoints.swift [hunk 1]
```diff
@@ -58,4 +58,20 @@ extension AirRobeEndpoint {
...
     }
 
+    static func identifyOrder(orderId: String) -> AirRobeEndpoint {
+        let requestBody: [String: Any] = [
+            "app_id": configuration?.appId ?? "",
+            "anonymous_id": UIDevice.current.identifierForVendor?.uuidString ?? "",
+            "session_id": sessionId,
+            "external_order_id": orderId,
+            "split_test_variant": "default",
+            "opted_in": UserDefaults.standard.OrderOptedIn
```
Comment (michaeldawson) [inline Sources/AirRobeWidget/Service/Endpoints/AirRobeWidgetEndpoints.swift:68]: I don't think `UserDefaults.standard.OrderOptedIn` should be stored as a constant. I think we should pass it in as an argument with `orderId`.

Comment (creative-dev-lab) [reply Sources/AirRobeWidget/Service/Endpoints/AirRobeWidgetEndpoints.swift:68]: Ok so like this?
```swift
static func identifyOrder(orderId: String, orderOptedIn: Bool) -> AirRobeEndpoint
```

And call like this 👇 
```swift
apiService.identifyOrder(orderId: orderId, orderOptedIn: UserDefaults.standard.OrderOptedIn)
```

File: Sources/AirRobeWidget/Service/Endpoints/AirRobeWidgetEndpoints.swift [hunk 2]
```diff
@@ -58,4 +58,20 @@ extension AirRobeEndpoint {
         )
     }
 
+    static func identifyOrder(orderId: String, orderOptedIn: Bool) -> AirRobeEndpoint {
```
Comment (creative-dev-lab) [inline Sources/AirRobeWidget/Service/Endpoints/AirRobeWidgetEndpoints.swift:61]: @michaeldawson 
Please check this whether it's what was expected.

Comment (michaeldawson) [reply Sources/AirRobeWidget/Service/Endpoints/AirRobeWidgetEndpoints.swift:61]: LGTM

**Review by michaeldawson — APPROVED** [review]

**Review by michaeldawson — APPROVED** [review]

**Pull request merged**